plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { SelectControl } from '@wordpress/components';
```

Comment:
We may want to use the new experimental `SelectControl` from `@woocommerce/components` for this so the dropdown supports typeahead field.
I am not sure how long a shipping class list could get, but being able to search for them in the field would be really nice.
cc: @jarekmorawski on this if we should use a standard dropdown for this, or a typeahead field?

Reply:
Because SelectControl is still experimental I will continue using the WP one until this SelectControl will become more stable. I left this [branch](https://github.com/woocommerce/woocommerce/tree/add/34328-shipping-class-autocomplete) with a partial implemetation that we can reuse for autocomplete implementation.